tests/sources/generate_diff.py

```diff
@@ -21,10 +21,18 @@
 import sys
 
 branch = os.getenv('TRAVIS_BRANCH')  # branch targeted by the pull request
-if branch is not None and os.getenv('TRAVIS_EVENT_TYPE') == 'pull_request':
+if branch is None or os.getenv('TRAVIS_EVENT_TYPE') != 'pull_request':
+    branch = os.getenv('APPVEYOR_REPO_BRANCH')
+    if os.getenv('APPVEYOR_PULL_REQUEST_NUMBER') is None:
```

Comment:
I needed it to be able to test this PR and didn't wanted to re-invent the wheel as it is already done in your PR. This will be automatically fixed when merging with your PR...